src/interface/ContextInterface.tsx

```diff
@@ -136,8 +146,11 @@ export async function retrieveVocabularyData(): Promise<boolean> {
     "OPTIONAL {?contextIRI rdfs:label ?label. }",
     "OPTIONAL {?contextIRI dcterms:title ?title. }",
     "graph ?contextIRI {",
-    "?vocab a <https://slovník.gov.cz/datový/pracovní-prostor/pojem/slovníkový-kontext> ",
-    "filter not exists {?vocab a <https://slovník.gov.cz/datový/pracovní-prostor/pojem/slovníkový-kontext-pouze-pro-čtení>.}",
+    "?vocab a ?vocabType.",
+    `values ?vocabType {<${[
+      parsePrefix("d-sgov-pracovní-prostor-pojem", "slovníkový-kontext"),
+      parsePrefix("a-popis-dat-pojem", "slovníkový-kontext"),
```

Comment:
I don't think that this is right - same argument as above. This could bring us a lot of problems.

Reply:
Restored back to dsgov in the newest commit.